CMakeLists.txt

```diff
@@ -334,18 +334,11 @@ include_directories(BEFORE ${OPENSSL_INCLUDE_DIR})
 # Make everyone find the highwayhash includes
 include_directories(BEFORE ${CMAKE_CURRENT_SOURCE_DIR}/auxil/highwayhash)
 
-# Alpine support
-if ( ${CMAKE_SYSTEM_NAME} MATCHES Linux AND EXISTS /etc/os-release )
-    execute_process(
-        COMMAND grep -q alpine /etc/os-release
-        RESULT_VARIABLE os_release_alpine
-    )
-
-    if ( os_release_alpine EQUAL 0 )
-        find_package(FTS REQUIRED)
-        list(APPEND OPTLIBS ${FTS_LIBRARY})
-        include_directories(BEFORE ${FTS_INCLUDE_DIR})
-    endif ()
+# Musl support
+if ( NEEDS_LIBFTS )
```

Comment:
Where is this variable defined? I see it defined in `srcpkgs/zeek/template` in the other project, but nowhere in Zeek.